releasenotes/notes/K8S-381_monitor_mk8s_containers-1a478428700d9039.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Managed Kubernetes instals several laxc-conatiners for various
```

Comment:
I didn't actually review this, I was just scrolling through quickly and my eyes caught this typo: s/instals/installs

Reply:
Also, is laxc a thing, or is it lxc?